packages/valory/skills/decision_maker_abci/behaviours/decision_request.py

```diff
@@ -75,9 +75,13 @@ def async_act(self) -> Generator:
         """Do the action."""
         with self.context.benchmark_tool.measure(self.behaviour_id).local():
             payload_content = None
+            mocking_mode: Optional[bool] = self.benchmarking_mode.enabled
```

Comment:
is the mocking_mode only active for the benchmarking_mode? Then why not giving the same name?

Reply:
:man_shrugging: